src/test/readme_examples.spec.ts

```diff
@@ -615,4 +615,126 @@ subscription {
             })
         })
     })
+
+    describe("Inline Fragments", () => {
```

Comment:
It is great to have these tests! Could they be added to the readme again? (I am starting to wonder whether there is an automated way to add these.)